src/page-content/navigation/header.json

```
@@ -61,22 +61,22 @@
       "mainMenu": [
         {
           "title": "Documentation",
-          "description": "Documentation for all Ory products",
+          "description": "Open Source Documentation",
```

```suggestion
          "description": "Open Source Documentation",
```

This is also the documentation for Ory Cloud! 
Revert this as it is misleading IMO.